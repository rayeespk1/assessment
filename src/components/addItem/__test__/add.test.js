import React from "react";
import '@testing-library/jest-dom'
import { render, cleanup } from "@testing-library/react";
import Add from "../add";

afterEach(cleanup);

test("renders", () => {
    const { asFragment } = render(<Add />);
    expect(asFragment()).toMatchSnapshot();
});

test("loading correct data", () => {
    const { getByTestId } = render(<Add />);
    expect(getByTestId("headingTag")).toHaveTextContent("Enter Item Details");
});