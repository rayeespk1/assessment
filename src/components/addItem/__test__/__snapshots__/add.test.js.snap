// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <section
    class="item-form"
  >
    <h3
      data-testid="headingTag"
    >
      Enter Item Details
    </h3>
    <form>
      <div
        class="Input"
      >
        <input
          class="InputElement"
          placeholder="Item Name"
          type="text"
          value=""
        />
      </div>
      <div
        class="Input"
      >
        <textarea
          class="InputElement"
          placeholder="Comments"
          type="text"
        />
      </div>
      <div
        class="close-container"
      >
        <p
          class="close-link"
        >
          Cancel
        </p>
        <button
          class="Button Success"
          data-testid="buttonTag"
        >
          SAVE
        </button>
      </div>
    </form>
  </section>
</DocumentFragment>
`;
